src/main/cpp/subsystems/RobotContainer.cpp

```diff
@@ -21,10 +20,17 @@ void RobotContainer::ConfigureButtonBindings() {
 }
 
 void RobotContainer::UpdateSensors() {
-  _ledController.Update();
+  
 }
 
 void RobotContainer::UpdateControls() {
+  if (_controlBoard.GetFlywheelPrepDesired()) {
```

Comment:
This should follow this scheduling trigger paradigm
https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html#whileactivecontinuous-whileheld

```
//create a specific button somewhere in controlboard
frc2::JoystickButton exampleButton(&exampleStick, 1);

// in 
exampleButton.whileHeld(shootCommand);
```

I don't know the actual syntax, but that should be close. I believe this happens during initialization rather than during the periodic call.